src/execution/execute.js

```diff
@@ -114,6 +118,7 @@ export type ExecutionContext = {|
 export type ExecutionResult = {|
   errors?: $ReadOnlyArray<GraphQLError>,
   data?: ObjMap<mixed> | null,
+  patches?: AsyncIterable<ExecutionPatchResult>,
```

Comment:
The result of `execute` (and also the main `graphql` function) has been modified to return a `patches` field, which is an `AsyncIterable`. We chose not to change the result of `execute` to something like `AsyncIterator<AsyncExecutionResult> | Promise<ExecutionResult>` as that would cause type errors for existing consumers of these functions, even if they do not enable these features in their schema.